oslabs-beta/GraphPulse
# Ds/right container

## Overview

**Issue Type**

[ ] Bug
[ x ] Feature
[ ] Tech Debt

**Description**
Filled out the Right Container with the basic components and containers needed for the app. Also added basic styling to space things out.

Installed d3 for future use.

**Ticket/Card Name**
SCRUM-3, SCRUM-5
	
**Steps to Validate Feature**

1. npm start or npm run dev
2. See if containers/components display properly on browser
